src/Quaternion.jl

```diff
@@ -236,6 +236,17 @@ function Base.isequal(q::Quaternion, w::Quaternion)
     isequal(q.s, w.s) & isequal(q.v1, w.v1) & isequal(q.v2, w.v2) & isequal(q.v3, w.v3)
 end
 
+if UInt === UInt64
```

Comment:
Would this be better?
```suggestion
@static if UInt === UInt64
```

Reply:
The `@static` macro brings better performance if it is used inside a function, but this case is not, so it seems there would not be better performance here.

```julia
julia> f() = @static Sys.iswindows() ? 1 : 2
f (generic function with 1 method)

julia> @code_lowered f()
CodeInfo(
1 ─     return 2
)

julia> g() = Sys.iswindows() ? 1 : 2
g (generic function with 1 method)

julia> @code_lowered g()
CodeInfo(
1 ─ %1 = Base.Sys.iswindows
│   %2 = (%1)()
└──      goto #3 if not %2
2 ─      return 1
3 ─      return 2
)
```